__tests__/app.test.js

```diff
@@ -412,3 +412,77 @@ describe("Post comment by articles id", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("should respond with updated article", () => {
+    const body = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(body)
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        const expected = {
+        article_id: 1,
+        title: 'Living in the shadow of a great man',
+        topic: 'mitch',
+        author: 'butter_bridge',
+        body: 'I find this existence challenging',
+        created_at: '2020-07-09T19:11:00.000Z',
+        votes: 105,
+        article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+      }
+        expect(typeof article).toBe("object");
+        expect(Array.isArray(article)).toBe(false);
+        expect(article).toEqual(expected)
+      });
+  });
+
+  test("should allow subtraction of votes", () => {
+    const body = { inc_votes: -10 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(body)
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article.votes).toBe(90);
+      });
+  });
+
+  test("should respond with 404 for valid but non existent article request", () => {
+    const body = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/1000")
+      .send(body)
+      .expect(404)
+      .then((response) => {
+        const { msg } = response.body;
+        expect(msg).toBe("article not found");
+      });
+  });
+
+  test("should respond with 400 error for invalid article request", () => {
+    const body = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/robot")
+      .send(body)
+      .expect(400)
+      .then((response) => {
+        const { msg } = response.body;
+        expect(msg).toBe("bad request");
+      });
+  });
+
+  test("should respond with 400 error for invalid update", () => {
+    const body = { inc_votes: "robot" };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(body)
+      .expect(400)
+      .then((response) => {
+        const { msg } = response.body;
+        expect(msg).toBe("bad request");
+      });
+  });
+});
```

Comment:
Great testing here! You may also look to add a test for Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)